testing/controller-test.js

```diff
@@ -0,0 +1,12 @@
+(function(){
```

Comment:
Don't merge code like this in real life - code to be merged should be production quality.